bundles/org.openhab.binding.nuvo/src/main/java/org/openhab/binding/nuvo/internal/handler/NuvoHandler.java

```diff
@@ -860,7 +880,18 @@ public void onNewMessageEvent(NuvoMessageEvent evt) {
                     updateChannelState(NuvoEnum.valueOf(ZONE + zoneId), CHANNEL_TYPE_LOUDNESS,
                             ONE.equals(matcher.group(4)) ? ON : OFF);
                 } else {
-                    logger.debug("no match on message: {}", updateData);
+                    matcher = ZONE_CFG_PATTERN.matcher(updateData);
+                    // example: ENABLE1,NAME"Great Room",SLAVETO0,GROUP1,SOURCES63,XSRC0,IR1,DND0,LOCKED0,SLAVEEQ0
+                    if (matcher.find()) {
+                        // Future; utilize other info such as zone name, available sources bitmask, etc.
```

Comment:
Could you change Future to TODO, it will then show up in SAT and will be on the 'radar' for future developments.